api/db/seeds/data/team-certification/shared/constants.js

```diff
@@ -16,18 +16,22 @@ export const CERTIFIABLE_SUCCESS_USER_ID = TEAM_CERTIFICATION_OFFSET_ID_USERS +
 const TEAM_CERTIFICATION_OFFSET_ID_CENTERS = TEAM_CERTIFICATION_OFFSET_ID + 300;
 export const PRO_CERTIFICATION_CENTER_ID = TEAM_CERTIFICATION_OFFSET_ID_CENTERS;
 export const SCO_CERTIFICATION_CENTER_ID = TEAM_CERTIFICATION_OFFSET_ID_CENTERS + 1;
+export const CLEA_CERTIFICATION_CENTER_ID = TEAM_CERTIFICATION_OFFSET_ID_CENTERS + 2;
 
 /// EXTERNAL IDS
 export const SHARED_ORGANIZATION_EXTERNAL_ID = 'SHARED_CERTIFICATION_ORGANIZATION';
 export const PRO_CERTIFICATION_CENTER_EXTERNAL_ID = 'PRO_CERTIFICATION_CENTER';
 export const SCO_CERTIFICATION_CENTER_EXTERNAL_ID = 'SCO_CERTIFICATION_CENTER';
+export const CLEA_V3_CERTIFICATION_CENTER_EXTERNAL_ID = 'CLEA_V3_CERTIFICATION_CENTER';
```

Comment:
```suggestion
export const PRO_WITH_CLEA_CERTIFICATION_CENTER_EXTERNAL_ID = 'PRO_WITH_CLEA_CERTIFICATION_CENTER';
```

**question:** a-t-on vraiment besoin de préciser `_V3_` dans les constantes CLEA ? 
Ça pourrait sous-entendre que la certif peut être passée sous plusieurs versions, or ce n'est pas le cas il me semble ?

Reply:
Je peux enlever le V3, surtout que effectivement je mentionne double certification donc c'est force v3.

Reply:
Done